pages/common/p7zip.md

```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper of 7-Zip file archiver with high compression ratio.
+> Internally executes either 7za or 7zr command.
```

Comment:
```suggestion
> Internally executes either 7za or 7zr command.
> More information: <http://p7zip.sourceforge.net>.
```

We could add the homepage here.

Reply:
I totally forgot about that :man_facepalming: